index.bs

```diff
@@ -277,9 +277,9 @@ partial interface Document {
 </pre>
 
 The {{pictureInPictureEnabled}} attribute's getter must return
-<code>true</code> if the <a>context object</a> is <a>allowed to use</a> the
-feature indicated by attribute name <code>picture-in-picture</code> and
-<a>Picture-in-Picture is supported</a>, and false otherwise.
+<code>true</code> if <a>Picture-in-Picture is supported</a> and the <a>context
+object</a> is <a>allowed to use</a> the feature indicated by attribute name
+<code>picture-in-picture</code>, and false otherwise.
```

Comment:
Isn't this about the feature policy? Does that mean that the steps above (checking for `pictureInPictureEnabled` would check for the feature policy too implicitly, making step 2 moot?

Reply:
You're absolutely right! Good catch.